### PR TITLE
feat(Icons): Add Gripper and Expand icons for Portfolio project UI

### PR DESCRIPTION
**Pull request by Kooshaba**
Ticket: [LX-4196](https://codecademy.atlassian.net/browse/LX-4196)
Design: https://www.figma.com/file/Mmoo9FiPHdfGbfODDr0Ydh/Portfolio-Projects?node-id=1732%3A1374

### CODE REVIEW

**Review by JoshuaKGoldberg — CHANGES_REQUESTED** [review]
All of our existing icons have `viewBox="0 0 24 24"`. These should also be, I think.

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.8.1-alpha.010a77.0<br/>@codecademy/gamut-labs@4.2.5-alpha.010a77.0<br/>@codecademy/gamut@16.0.15-alpha.010a77.0<br/></pre>

**Review by JoshuaKGoldberg — CHANGES_REQUESTED** [review]
We chatted a bit on Slack: the Expand icon should be scaled a bit so it fits its icon area. The Gripper really isn't a square so let's try out a non-square viewport for it? I couldn't find any breakages in a `yarn start` locally. 🚀 

fyi @codecaaron

File: packages/gamut-icons/src/svg/gripper-icon.svg [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (casey-speer) [inline packages/gamut-icons/src/svg/gripper-icon.svg:1]: Curious if the gripper glyph is something we'll be reusing or if it's more of a one-off that could live alongside the kanban board — what do you think @JoshuaKGoldberg @Kooshaba ?

Comment (JoshuaKGoldberg) [reply packages/gamut-icons/src/svg/gripper-icon.svg:1]: I'll defer to design on this one.

**Comment by codecaaron** [comment]
Is there a reason we are not using `FullscreenIcon` for the expand?

File: packages/gamut-icons/src/svg/expand-icon.svg [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <path d="M7 10l-6 6M16 5.5V1h-4.5M1 11.5V16h4.5M16 1l-6 6" stroke="currentColor" stroke-linecap="round"
+    stroke-linejoin="round" />
+</svg>
```
Comment (codecaaron) [inline packages/gamut-icons/src/svg/expand-icon.svg:4]: Isn't this covered by `FullscreenIcon`?

**Comment by codecaaron** [comment]
@Kooshaba are we good to close this per our conversation about the other icons available?

**Comment by codecaaron** [comment]
@Kooshaba closing this in favor of current solution.

**Pull request closed without merging**